sqlglot/parser.py

```diff
@@ -8244,3 +8247,11 @@ def _parse_max_min_by(self, expr_type: t.Type[exp.AggFunc]) -> exp.AggFunc:
         return self.expression(
             expr_type, this=seq_get(args, 0), expression=seq_get(args, 1), count=seq_get(args, 2)
         )
+
+    def _identifier_expression(
+        self, token: t.Optional[Token] = None, **kwargs: t.Any
+    ) -> exp.Identifier:
+        token = token or self._prev
+        expression = self.expression(exp.Identifier, this=token.text, **kwargs)
+        expression.update_positions(token)
```

Comment:
If an identifier is quoted, we probably want to include the quotes so that a consumer of the line/col information can re-parse that identifier and respect its case-sensitivity.

I'd suggest adding some tests that include quotes as well.

Reply:
Added some tests for quoted identifiers